modules/oe_whitelabel_helper/src/Plugin/Block/CorporateEuLogoBlock.php

```diff
@@ -83,19 +84,21 @@ public function build(): array {
     $image = [
       '#theme' => 'image',
       '#uri' => $logo_path . '/logo-eu--' . $language . '.svg',
-      '#width' => '240px',
-      '#height' => '60px',
       '#alt' => $title,
       '#title' => $title,
     ];
-    $mobile_url = file_create_url($logo_path . '/mobile/logo-eu--' . $language . '.svg');
+
     $build = [
-      '#type' => 'inline_template',
-      '#template' => '<picture><source media="(max-width: 25em)" srcset="{{ mobile }}">{{ image }}</picture>',
-      '#context' => [
-        'image' => $image,
-        'mobile' => file_url_transform_relative($mobile_url),
-      ],
+      '#type' => 'link',
+      '#url' => Url::fromUri('https://europa.eu/european-union/index_' . $language, [
```

Comment:
Same as above. Would be good to consider to use a path set at config and not hard-coded. Let's see if this can be set as config.